### DISCUSSION TITLE
Question: how to search in the tree?

### DISCUSSION THREAD

**Discussion by mix-7** [localauthor/org-tree#7] [General]
Question: how to search in the tree?

Is there a search in the tree for one or more words?
Filter by keywords.
If not, can you implement it?

Thanks!

**Comment by localauthor** [comment]
Any search function should work in the buffer. I use `isearch` and `consult-line`.

Is something not working as expected?

**Comment by mix-7** [comment]
Thanks, I'll try

**Comment by mix-7** [comment]
The tree is a separate buffer?
